packages/@dcl/inspector/src/lib/data-layer/host/utils/migrations/fix-network-entity-values.ts

```diff
@@ -0,0 +1,35 @@
+import { Entity, IEngine } from '@dcl/ecs'
+import { CoreComponents, SdkComponents } from '../../../../sdk/components'
+import { INSPECTOR_ENUM_ENTITY_ID_START } from '../../../../sdk/enum-entity'
+
+/**
+ * Fixes duplicate entity IDs in NetworkEntity components within the engine.
+ *
+ * This function scans through all entities that have the NetworkEntity component
+ * and identifies any duplicate entity IDs. If duplicates are found, it reassigns
+ * unique entity IDs to each NetworkEntity component to ensure all entity IDs are unique.
+ */
+export function fixNetworkEntityValues(engine: IEngine) {
```

Comment:
will this logic re-assign a new entityId for all the entities if one of them is duplicated?

Reply:
YES